ui/app/models/ldap/config.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import { withModelValidations } from 'vault/decorators/model-validations';
+
+const validations = {
```

Comment:
I honestly can't keep track of when or when we do not capitalize names. But do the designs have it as all three of these words being capitalized (Administrator Distinguished Name)? I'll go with whatever design has, but I'm used to only capitalizing the first word and not any following. Ex: Key type or Maximum versions even though the param is max_versions. Same goes for Administrator Password.

Reply:
I took the labels straight from the designs so made them the same in these messages but I can change them.